packages/web/src/components/srcbook-cards.tsx

```diff
@@ -242,7 +242,7 @@ export function ImportSrcbookCTA(props: { onClick: () => void }) {
   return (
     <CardContainer
       onClick={props.onClick}
-      className="w-full sm:w-[214px] sm:max-w-[214px] bg-muted hover:border-foreground focus-within:border-foreground focus-within:ring-1 focus-within:ring-foreground active:translate-y-0.5"
+      className="w-full sm:w-[214px] sm:max-w-[214px] bg-muted border-dashed hover:border-solid hover:border-foreground focus-within:border-foreground focus-within:ring-1 focus-within:ring-foreground active:translate-y-0.5"
```

Comment:
We can't use `dashed` because it is inconsistent with the dashes we use from the designs (Justin's svg). This looks bad with two different dashes

Reply:
I know it's inconsistent but I think it looks better than not dashed, this was a personal taste call:
1. They both don't look great
2. Dashed at least conveys the right UX, so it wins out

Reply:
the solid border looks fine to me and it we use it elsewhere. What is the "right ux" ?

Reply:
Noticed a few times now that we just have different taste for some small things. Not very productive.

Right UX = things with dashed borders imply you can drag things into them